Sources/MongoSwift/MongoClient.swift

```diff
@@ -402,6 +405,106 @@ public class MongoClient {
         )
     }
 
+#if compiler(>=5.5.2) && canImport(_Concurrency)
+    @available(macOS 10.15, *)
+    internal class CmdHandler: CommandEventHandler {
+        private var continuation: AsyncStream<CommandEvent>.Continuation
+        internal init(continuation: AsyncStream<CommandEvent>.Continuation) {
+            self.continuation = continuation
+        }
+
+        // Satisfies the protocol
+        internal func handleCommandEvent(_ event: CommandEvent) {
+            self.continuation.yield(event)
+        }
+
+        internal func finish() {
+            self.continuation.finish()
+        }
+    }
+
+    @available(macOS 10.15, *)
+    internal class SDAMHandler: SDAMEventHandler {
+        private var continuation: AsyncStream<SDAMEvent>.Continuation
+        internal init(continuation: AsyncStream<SDAMEvent>.Continuation) {
+            self.continuation = continuation
+        }
+
+        // Satisfies the protocol
+        internal func handleSDAMEvent(_ event: SDAMEvent) {
+            self.continuation.yield(event)
+        }
+
+        internal func finish() {
+            self.continuation.finish()
+        }
+    }
+
+    /// Provides an `AsyncSequence` API for consuming command monitoring events.
+    /// Example: printing the command events out would be written as
```

Comment:
Using a block comment for the example code would be a bit more readable than just inline.
e.g.
```swift
for await event in client.commandEventStream() {
    print(event)
}
```
Ditto for the SDAM event example.

Reply:
So in order the code to be formatted as code, you need the triple backticks (\`\`\`) like in markdown. I think you also need to keep prepending the line with `*`. For an existing example, check out [ClientSession.swift](https://github.com/mongodb/mongo-swift-driver/blob/main/Sources/MongoSwift/ClientSession.swift#L17).

Reply:
As per discussion, this method works but used the backticks method for codebase consistency